wasm/bindings/TranslationModelBindings.cpp

```diff
@@ -6,17 +6,23 @@
 
 #include <emscripten/bind.h>
 
-#include "TranslationModel.h"
+#include "response.h"
+#include "service.h"
 
 using namespace emscripten;
 
+typedef marian::bergamot::Service TranslationModel;
+typedef marian::bergamot::Response TranslationResult;
+
 // Binding code
 EMSCRIPTEN_BINDINGS(translation_model) {
   class_<TranslationModel>("TranslationModel")
-    .constructor<std::string>()
-    .function("translate", &TranslationModel::translate)
-	  .function("isAlignmentSupported", &TranslationModel::isAlignmentSupported)
-    ;
+      .constructor<std::string>()
+      .function("translate", &TranslationModel::translateMultiple)
+      .function("isAlignmentSupported",
+                &TranslationModel::isAlignmentSupported);
+  // ^ We redirect translateMultiple to translate instead. Sane API is
```

Comment:
What are these comments?

Reply:
So the original thing I was told was to have `std::future<Response>` and async operations. `TranslationModel` changes these to a blocking API where they instead feed us multiple texts. I have asked why this because I can't infer from signature alone, now I do guesswork and I read this as "browser needs further granularity in texts but same speed".

So the disparity is:
```haskell
f :: InputType -> OutputType

Service::translate :: Str -> Future[Response]
TranslationModel::translate :: Vector[Str] -> Vector[Response]
```

To arrange parity, the following is what I did:

```haskell
Service::translateMultiple:: Vector[Str] -> Vector[Response]
```

I consider the async future to be the sane API from the feedback, input I have received so far. At wasm bindings, I'm binding `translateMultiple` instead to "translate" and plumbing to test compatibility at extension.